### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by abhidg**
- Report carbon intensity in human readable format #92
- Report carbon intensity in human-readable format #92
- Show friendlier error when at(1) is missing
- Show error if duration is greater than 48 hours

### CODE REVIEW

**Review by andreww — APPROVED** [review]
Looks good to me - just minor comments. I think the reduction in test coverage reported by codecov probably isn't a problem. Can we tone back the settings so it provides information rather than blocking merging?

File: cats/__init__.py [hunk 1]
```diff
@@ -287,7 +308,9 @@ def main(arguments=None) -> int:
     else:
         print(output)
     if args.command and args.scheduler == "at":
-        schedule_at(output, args.command.split())
+        if err := schedule_at(output, args.command.split()):
```
Comment (abhidg) [inline cats/__init__.py:311]: Failure modes are tracked by individual unit tests

File: tests/test_main.py [hunk 2]
```diff
@@ -40,6 +40,13 @@ def test_schedule_at(fp):
     )
 
 
+def test_schedule_at_missing():
```
[the remark this answers is not in the document]
Comment (abhidg) [reply tests/test_main.py:43]: Added a test!

File: cats/__init__.py [hunk 3]
```diff
@@ -169,14 +169,16 @@ class CATSOutput:
...
+Carbon intensity at optimal time          = {self.carbonIntensityOptimal.value:.2f} gCO2eq/kWh"""
 
         if self.emmissionEstimate:
-            out += (
-                f"\nEstimated emmissions for running job now: {self.emmissionEstimate.now}\n"
-                f"Estimated emmissions for running delayed job: {self.emmissionEstimate.best}"
-                f" (- {self.emmissionEstimate.savings})"
-            )
+            out += f"""
+Estimated emissions if job started now    = {self.emmissionEstimate.now}
```
[the remark this answers is not in the document]
Comment (abhidg) [reply cats/__init__.py:201]: probably gCO2 - will check and put in another PR for that

**Pull request merged**